README.md

```diff
@@ -1,84 +1,124 @@
 
-**Screenshot**: If the software has visual components, place a screenshot after the description; e.g.,
+### Persistence
 
-![](https://raw.githubusercontent.com/cfpb/open-source-project-template/master/screenshot.png)
+Module responsible for persisting information into the system. It becomes the system of record for HMDA data
+
+### API
+
+This module contains both public APIs for HMDA data for general use by third party clients and web applications, as well as endpoints for receiving data and providing information about the filing process for Financial Institutions
 
 
 ## Dependencies
 
-Describe any dependencies that must be installed for this software to work.
-This includes programming languages, databases or other storage mechanisms, build tools, frameworks, and so forth.
-If specific versions of other software are required, or known not to work, call that out.
+### Java 8 SDK
+
+The HMDA Platform runs on the Java Virtual Machine (JVM), and requires the Java 8 JDK to build and run the project. This project is currenly being built and tested on [Oracle JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html). See [Oracle's JDK Install Overview](http://docs.oracle.com/javase/8/docs/technotes/guides/install/install_overview.html) for install instructions.
+
+The HMDA Platform should also run on OpenJDK 8.
+
+### Scala
+
+The HMDA Platform is written in [Scala](http://www.scala-lang.org/). To build it, you will need to [download](http://www.scala-lang.org/download/) and [install](http://www.scala-lang.org/download/install.html) Scala 2.11.x
+
+In addition, you'll need Scala's interactive build tool [sbt](http://www.scala-sbt.org/0.13/tutorial/index.html). Please refer to sbt's [installation instructions](http://www.scala-sbt.org/0.13/tutorial/Setup.html) to get started.
 
-## Installation
 
-Detailed instructions on how to install, configure, and get the project running.
-This should be frequently tested to ensure reliability. Alternatively, link to
-a separate [INSTALL](INSTALL.md) document.
+## Building and Running
 
-## Configuration
+The HMDA Platform uses sbt's multi-project builds, each project representing a specific task.
 
-If the software is configurable, describe it in detail, either here or in other documentation to which you link.
+### Interactive
 
-## Usage
+1. Start `sbt`
 
-Show users how to use the software.
-Be specific.
-Use appropriate formatting when showing code snippets.
+```shell
+$ sbt
+```
 
-## How to test the software
+2. Select project to build and run.This will retrieve all necessary dependencies, compile Scala source, and start a local server. It also listens for changes to underlying source code, and auto-deploys to local server.
 
-If the software includes automated tests, detail how to run those tests.
+```shell
+> projects
+[info] In file:/Users/Juan/Development/hmda-platform/
+[info] 	 * hmda
+[info] 	   model
+[info] 	   parser
 
-## Known issues
+> re-start
```

Comment:
The `~` fell off the `~re-start` with that last commit.  Without it, we won't get the _listens for changes to underlying source code, and auto-deploys to local server_ mentioned above.

* https://github.com/spray/sbt-revolver#triggered-restart